lib/pgBackRest/Db.pm

```diff
@@ -90,7 +92,12 @@ sub new
 
         if (!isDbLocal({iRemoteIdx => $self->{iRemoteIdx}}))
         {
-            $self->{oProtocol} = protocolGet(CFGOPTVAL_REMOTE_TYPE_DB, $self->{iRemoteIdx});
+            eval
+            {
+                $self->{oProtocol} = protocolGet(CFGOPTVAL_REMOTE_TYPE_DB, $self->{iRemoteIdx}, {bWarnOnError => $bWarnOnError});
+                return true;
+            }
+            or do {};
         }
```

Comment:
The eval is in "if (!isDbLocal" condition, so I'm not sure I'm following when you say "it will try to connect to local host"